Bottom: aa120e1a6260a40d56958fe39fd3a9e50ffbeb38
Top:    b18f3aa1e9afeccf53057e54182e33c3db0df2ca
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:12:20 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index f6c714b..8ad94c5 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
